src/main/scala/com/atomist/rug/test/gherkin/project/ProjectScenarioWorld.scala

```diff
@@ -71,6 +72,11 @@ class ProjectScenarioWorld(
     project.updateTo(resultAs)
   }
 
+  // for calling from nashorn which doesn't like default parameter values!
+  def generateWith(generator: ProjectGenerator, projectName: String): Unit = {
```

Comment:
I remember @alankstewart having an issue with either methods having default parameter values or overloaded methods when crossing between Nashorn and Scala. I guess this is the workaround?